### ISSUE TITLE
As a user, I should be able to select an entity color when creating Schemas and Types so that diagrams can be more easily followed

### ISSUE THREAD

**Issue by damusix** [design-first/system-designer#149]
**Is your feature request related to a problem? Please describe.**
Problem is, diagrams get very wiry and hard to follow.

**Describe the solution you'd like**
It would be nice if a color picker could be included to highlight entities (background and or text and or border, etc)

**Describe alternatives you've considered**
Using System Designer as a playground and finalizing my UML using another software, or maybe Draw.io

**Comment by ecarriou** [comment]
Hello,

Thank you for your suggestion !
Could you tell me where do you want to add the color ? When we show the full diagram of the system ?
